cloud_AWS/terraform/module/examples/single-vpc/main.tf

```diff
@@ -6,21 +6,28 @@ terraform {
     }
     kentik-cloudexport = {
       version = ">= 0.2.0"
-      source = "kentik/kentik-cloudexport"
+      source  = "kentik/kentik-cloudexport"
     }
   }
 }
 
 provider "aws" {
   region = "us-east-2"
-}
-
-provider "kentik-cloudexport" {
 
+  # Allow module to assume role with external ID
+  //  assume_role {
+  //    role_arn     = "arn:aws:iam::ACCOUNT_ID:role/ROLE_NAME"
+  //    external_id = "a43dd248-0047-11ec-ab2e-6b5423aa2e8a"
```

Comment:
External ID, in Kentik's case, is the unique company ID, which looks more like an integer than UID. Users actually cannot choose the `external id`, because there is no way to share it with the Transfluo process that actually does the "assume role" AWS API call. So, the example needs to guide users to fill-in their Kentik company ID. The only way to obtain the company ID in Kentik UI is to start "automated configuration" of AWS cloudexport (https://portal.kentik.com/v4/setup/clouds/aws). This fills the company ID as external_id in the Terraform resource file :).

Reply:
Improved variable comments and added a comment in the example